syntax = "proto3";
package CoordinateViewer;

message ReturnStatus {}

message CallPoint {
    string  number = 1;
    string  imei = 2;
    float   lat = 3;
    float   lon = 4;
}

message Session {
    fixed64             timestamp = 1;
    fixed64             duration = 2;
    repeated CallPoint  points = 3;
}

service CoordinateViewerService {
    rpc add_session(Session) returns (ReturnValue) {}
}
